docs/pages/database-access/guides/redis-aws.mdx

```diff
@@ -5,6 +5,11 @@ description: How to configure Teleport database access with AWS ElastiCache and
 
 (!docs/pages/includes/database-access/db-introduction.mdx  dbType="AWS ElastiCache and MemoryDB for Redis cluster" dbConfigure="AWS ElastiCache and MemoryDB for Redis cluster database with IAM authentication" dbName="AWS ElastiCache and MemoryDB for Redis" !)
 
+## How it works
+
+(!docs/pages/includes/database-access/how-it-works/iam.mdx db="ElastiCache or
+MemoryDB for Redis cluster" cloud="AWS"!)
```

Comment:
We have two modes for AWS redis
- IAM auth for Redis 7.0+
- Teleport-managed users using Secrets manager

Details see https://goteleport.com/docs/database-access/guides/redis-aws/#step-56-configure-authentication-for-elasticache-or-memorydb-users